.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -91,33 +134,105 @@ jobs:
+
+      - name: Echo user subjects used for this test
+        run: |
+          echo ${{ env.USER_SUBJECTS }}
+
       - name: dispatch to terra-github-workflows
         env:
           rawls_test_command: "testOnly -- -l ProdTest -l NotebooksCanaryTest -n org.broadinstitute.dsde.test.api.WorkspacesAzureTest"
         uses: broadinstitute/workflow-dispatch@v3
         with:
           workflow: .github/workflows/rawls-swat-tests.yaml
           repo: broadinstitute/terra-github-workflows
-          ref: refs/heads/main
+          ref: refs/heads/iv-az-e2e-1
+          token: ${{ env.TOKEN }}
+          inputs: '{
+            "run-name": "${{ env.RUN_NAME }}",
+            "bee-name": "${{ env.BEE_NAME }}",
+            "ENV": "qa",
+            "ref": "refs/heads/${{ needs.init-github-context.outputs.branch }}",
+            "test-group-name": "workspaces_azure",
+            "test-command": "${{ env.rawls_test_command }}",
+            "java-version": "17",
+            "billing-project": "${{ needs.params-gen.outputs.project-name }}",
+            "pipeline-inject": "RAWLS_AZURE_E2E",
+            "user-subjects": ${{ toJson(env.USER_SUBJECTS) }}
+          }'
+
+  delete-billing-project-v2-from-bee-workflow:
+    runs-on: ubuntu-latest
+    needs: [init-github-context, params-gen, rawls-swat-e2e-test-job]
+    if: false
+    steps:
+      - name: dispatch to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: .github/workflows/delete-billing-project-v2-from-bee.yaml
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/iv-az-e2e-1
           token: ${{ env.TOKEN }}
-          inputs: '{ "bee-name": "${{ env.BEE_NAME }}", "ENV": "qa", "ref": "refs/heads/${{ needs.init-github-context.outputs.branch }}", "test-group-name": "workspaces_azure", "test-command": "${{ env.rawls_test_command }}", "java-version": "17" }'
+          inputs: '{
+            "run-name": "${{ env.RUN_NAME }}",
+            "bee-name": "${{ env.BEE_NAME }}",
+            "billing-project": "${{ needs.params-gen.outputs.project-name }}",
+            "billing-project-owner": "${{ needs.init-github-context.outputs.owner-subject }}",
+            "service-account": "${{ needs.init-github-context.outputs.service-account }}"
+          }'
 
   destroy-bee-workflow:
     runs-on: ubuntu-latest
-    needs: [rawls-swat-e2e-test-job, init-github-context]
+    needs: [init-github-context, rawls-swat-e2e-test-job, delete-billing-project-v2-from-bee-workflow]
     if: ${{ needs.init-github-context.outputs.delete-bee && always() }} # always run to confirm bee is destroyed unless explicitly requested not to
-    permissions:
-      contents: 'read'
-      id-token: 'write'
     steps:
       - name: dispatch to terra-github-workflows
         uses: broadinstitute/workflow-dispatch@v3
```

Comment:
while we're here, can we add `wait-for-completion: false` to destroy bee? ultimately, we don't need to pay attention to this as this would get cleaned up by devops tooling in case of failure

Reply:
Added `wait-for-completion: false`